SR2020/2020-03-16-competition-postpone.md

```diff
@@ -0,0 +1,35 @@
+---
+to: sr2020-teams
+subject: Postponing Student Robotics 2020
```

Comment:
perhaps be clear that this is the competition event that's being postponed? Not sure if that matters here, since the competition year has already started.

Reply:
I too was wondering this, yea can't hurt!